### DISCUSSION TITLE
Events Made Easy Frontend Submit - Problem with some fields

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#393] [General]
Hi !
I'm trying to make working the Frontend Submit.
I've added these hidden fields within the forms and they don't seem to all work

<?php EMEFS::field('event_status','hidden',"value=1"); ?>
<?php EMEFS::field('event_rsvp','hidden',"value=1"); ?>
<?php EMEFS::field('location_id','hidden',"value=5"); ?>
<?php EMEFS::field('event_author','hidden',"value=4"); ?>
<?php EMEFS::field('event_contactperson_id','hidden',"value=4"); ?>

When I submit,  the event is created, is activated,  rsvp is enabled with event_seats field, but author is not correctly applied, location_id is not set and finally event_contactperson_id is not set.

What i'm doing wrong ?

**Comment by WXQ8NPb5TrbDNqzywcxF** [comment]
Sorry code did not paste

`<?php EMEFS::field('event_status','hidden',"value=1"); ?>

        <?php EMEFS::field('event_rsvp','hidden',"value=1"); ?>
        <?php EMEFS::field('location_id','hidden',"value=5"); ?>
        <?php EMEFS::field('event_author','hidden',"value=4"); ?>
        <?php EMEFS::field('event_contactperson_id','hidden',"value=4"); ?>`

**Reply by liedekef** [reply]
This is not the forum for frontend-submit. Please go to https://wordpress.org/support/plugin/events-made-easy-frontend-submit/ 
Also: hidden was never advertised for EMEFS::field (only for attributes and properties). The dev-version now works ok for EMEFS::field and hidden too, see https://wordpress.org/plugins/events-made-easy-frontend-submit/advanced/

**Comment by WXQ8NPb5TrbDNqzywcxF** [comment]
Thanks, i'm sorry to have posted on wrong place.  I'll have to make deeper test since my first test did not work with the dev version.
Thanks for your help.

**Reply by liedekef** [reply]
indeed, I forgot a "break" in the dev-version. Has been corrected, so should work now
